pkg/reconciler/managed/reconciler.go

```diff
@@ -675,14 +764,21 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		"external-name", meta.GetExternalName(managed),
 	)
 
+	managementPoliciesEnabled := r.features.Enabled(feature.EnableAlphaManagementPolicies)
+	if managementPoliciesEnabled {
+		log.WithValues("managementPolicies", managed.GetManagementPolicy())
+	}
+	actionChecker := NewManagementPolicyActionChecker(managementPoliciesEnabled, managed.GetManagementPolicy())
+
 	// Check the pause annotation and return if it has the value "true"
-	// after logging, publishing an event and updating the SYNC status condition
-	if meta.IsPaused(managed) {
-		log.Debug("Reconciliation is paused via the pause annotation", "annotation", meta.AnnotationKeyReconciliationPaused, "value", "true")
-		record.Event(managed, event.Normal(reasonReconciliationPaused, "Reconciliation is paused via the pause annotation"))
+	// or if the management policies are enabled and the ManagementPolicy is empty.
+	// Log, publish an event and update the SYNC status condition.
+	if IsPaused(managed, managementPoliciesEnabled) {
+		log.Debug("Reconciliation is paused through the management policy or the pause annotation")
+		record.Event(managed, event.Normal(reasonReconciliationPaused, "Reconciliation is paused"))
```

Comment:
These might be more helpful if we named the annotation explicitly. Naming the management policy might be a bit verbose - perhaps we could at least say `spec.managementPolicy` to make it easier to find?